src/components/Legend/index.js

```diff
@@ -1,29 +1,17 @@
 import { StyledLegend, LegendLabel, Dino, LegendGroup } from './styled';
-import { ReactComponent as OnePointRedDino } from '../../icons/dinosaurs/red-one-point.svg';
-import { ReactComponent as TwoPointsRedDino } from '../../icons/dinosaurs/red-two-points.svg';
-import { ReactComponent as FivePointsRedDino } from '../../icons/dinosaurs/red-five-points.svg';
-import { ReactComponent as TenPointsRedDino } from '../../icons/dinosaurs/red-ten-points.svg';
+import { dinos, legends } from '../Level/staticLevels'
 
-const Legend = ({ color, reversed }) => {
+const Legend = ({ color, reversed, teamId }) => {
```

Comment:
I think you can rename `teamId` to `legend` here. (Don't forget to update it in other places)

Reply:
Agreed